exp/services/recoverysigner/internal/db/dbmigrate/dbmigrate_test.go

```diff
@@ -0,0 +1,239 @@
+package dbmigrate
+
+import (
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+
+	assetfs "github.com/elazarl/go-bindata-assetfs"
+	migrate "github.com/rubenv/sql-migrate"
+	"github.com/shurcooL/httpfs/filter"
+	dbpkg "github.com/stellar/go/exp/services/recoverysigner/internal/db"
+	"github.com/stellar/go/support/db/dbtest"
+	supportHttp "github.com/stellar/go/support/http"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGeneratedAssets(t *testing.T) {
+	localAssets := http.FileSystem(filter.Keep(http.Dir("."), func(path string, fi os.FileInfo) bool {
+		return fi.IsDir() || strings.HasSuffix(path, ".sql")
+	}))
+	generatedAssets := &assetfs.AssetFS{Asset: Asset, AssetDir: AssetDir, AssetInfo: AssetInfo}
+
+	if !supportHttp.EqualFileSystems(localAssets, generatedAssets, "/") {
+		t.Fatalf("generated migrations does not match local migrations")
+	}
+}
+
+func TestPlanMigration_upApplyOne(t *testing.T) {
+	db := dbtest.Postgres(t)
+	session, err := dbpkg.Open(db.DSN)
+	require.NoError(t, err)
+
+	migrations, err := PlanMigration(session, migrate.Up, 1)
+	require.NoError(t, err)
+	wantMigrations := []string{"20200309000000-initial-1.sql"}
+	assert.Equal(t, wantMigrations, migrations)
+}
+
+func TestPlanMigration_upApplyAll(t *testing.T) {
+	db := dbtest.Postgres(t)
+	session, err := dbpkg.Open(db.DSN)
+	require.NoError(t, err)
+
+	migrations, err := PlanMigration(session, migrate.Up, 0)
+	require.NoError(t, err)
+	require.Greater(t, len(migrations), 1)
+	assert.Equal(t, "20200309000000-initial-1.sql", migrations[0])
```

Comment:
I think you could also test for `migrations[1]`, like in lines [R92-R95](https://github.com/stellar/go/pull/2374/files#diff-1e33b298fa035a5984c3a09eac9f0ed8R92-R95):
```suggestion
	assert.Equal(t, "20200309000000-initial-1.sql", migrations[0])
	assert.Equal(t, ""20200309000001-initial-2.sql"", migrations[1])
```

Reply:
Good point. I did it slightly differently but the same result, see 1504a71.